Cloud Token Theft - XSOAR Version (#27808)
demisto/content: Packs/CloudIncidentResponse/Playbooks/playbook-Cortex_XDR_-_XCloud_Token_Theft_Response_README.md

---

## Cloud Token Theft Response Playbook

The **Cloud Token Theft Response Playbook** provides a structured and comprehensive flow to effectively respond to and mitigate alerts involving the theft of cloud tokens. The playbook supports AWS, GCP, and Azure and executes the following:

**Cloud Enrichment:**

> - Enriches the involved resources

Reviewer: ```suggestion
- Enriches the involved resources.
```

> - Enriches the involved identities

Reviewer: ```suggestion
- Enriches the involved identities.
```

> - Enriches the involved IPs

Reviewer: ```suggestion
- Enriches the involved IPs.
```

**Verdict Decision Tree:**

> - Determines the appropriate verdict based on the investigation findings

Reviewer: ```suggestion
- Determines the appropriate verdict based on the investigation findings.
```

**Early Containment using the Cloud Response - Generic Playbook:**

> - Implements early containment measures to prevent further impact

Reviewer: ```suggestion
- Implements early containment measures to prevent further impact.
```

**Cloud Persistence Threat Hunting:**

> - Conducts threat hunting activities to identify any cloud persistence techniques

Reviewer: ```suggestion
- Conducts threat hunting activities to identify any cloud persistence techniques.
```

**Enriching and Responding to Hunting Findings:**

> - Performs additional enrichment and responds to the findings from threat hunting

Reviewer: ```suggestion
- Performs additional enrichment and responds to the findings from threat hunting.
```

**Verdict Handling:**

> - Handles false positives identified during the investigation

Reviewer: ```suggestion
- Handles false positives identified during the investigation.
```

> - Handles true positives by initiating appropriate response actions

Reviewer: ```suggestion
- Handles true positives by initiating appropriate response actions.
```

| Remote usage of an AWS ECS token                    | AWS   |
| Suspicious usage of AWS service token               | AWS   |
| Remote usage of an App engine Service Account token | GCP   |
| Suspicious usage of App engine Service Account token| GCP   |
| Remote usage of VM Service Account token            | GCP   |
| Suspicious usage of VM Service Account toke         | GCP   |

---

## Dependencies

This playbook uses the following sub-playbooks, integrations, and scripts.

### Sub-playbooks

* TIM - Indicator Relationships Analysis
* IP Enrichment - Generic v2
* Cloud Response - Generic
* Entity Enrichment - Generic v3
* Cloud Threat Hunting - Persistence
* Cloud Enrichment - Generic
* Cortex XDR - XCloud Token Theft - Set Verdict
* Handle False Positive Alerts

### Integrations

This playbook does not use any integrations.

### Scripts

* ParseHTMLIndicators
* LoadJSON

### Commands

* xdr-get-cloud-original-alerts
* setIncident
* closeInvestigation

## Playbook Inputs

---

| **Name** | **Description** | **Default Value** | **Required** |
| --- | --- | --- | --- |
| alert_id | The alert ID. | alert.investigationId | Optional |

> | InternalRange | A list of internal IP ranges to check IP addresses against. The list should be provided in CIDR notation, separated by commas. |  | Optional |

Reviewer: ```suggestion
| InternalRange | A comma-separated list of internal IP ranges to check IP addresses against. The list should be provided in CIDR notation. |  | Optional |
```

| ResolveIP | Determines whether to convert the IP address to a hostname using a DNS query \(True/ False\). | True | Optional |
| earlyContainment | Whether to execute early containment.<br/>This action allows you to respond rapidly but have higher probability for false positives. | True | Optional |

> | VPNIPList | This input can process to types of data:<br/>1. A comma separated list of internal IPs assigned by the VPN provider using a XSIAM list or an hardcoded array.<br/>2. A link to an IP list which will be processed and extract the IP dynamically which each execution.<br/><br/>For CIDRs please use the InternalRange input. |  | Optional |

Reviewer: ```suggestion
| VPNIPList | This input can process two types of data:<br/>1. A comma-separated list of internal IPs assigned by the VPN provider using a XSIAM list or an hardcoded array.<br/>2. A link to an IP list which will be processed and extract the IP dynamically which each execution.<br/><br/>For CIDRs, use the InternalRange input. |  | Optional |
```
